src/classes/metadata/StudyMetadata.js

```diff
@@ -635,7 +635,17 @@ function _getDisplaySetFromSopClassModule(
     headers,
   });
 
-  return plugin.getDisplaySetFromSeries(series, study, dicomWebClient, headers);
+  let displaySet = plugin.getDisplaySetFromSeries(
+    series,
+    study,
+    dicomWebClient,
+    headers
+  );
+  if (displaySet) {
```

Comment:
can you make this `if (displaySet && !displaySet.modality)`, so we don't overwrite what the plugin returned?

Reply:
Sure, I will change it here